src/Microsoft.Tye.Extensions/Dapr/DaprExtension.cs

```diff
@@ -51,12 +106,12 @@ public override Task ProcessAsync(ExtensionContext context, ExtensionConfigurati
 
                         // These environment variables are replaced with environment variables
                         // defined for this service.
-                        Args = $"-app-id {project.Name} -app-port %APP_PORT% -dapr-grpc-port %DAPR_GRPC_PORT% --dapr-http-port %DAPR_HTTP_PORT% --metrics-port %METRICS_PORT% --placement-address localhost:50005",
+                        Args = $"-app-id {project.Name} -app-port %APP_PORT% -dapr-grpc-port %DAPR_GRPC_PORT% --dapr-http-port %DAPR_HTTP_PORT% --metrics-port %METRICS_PORT% --placement-address localhost:{placementPort}",
```

Comment:
That would be great this issue #802 also could be addressed here.

Reply:
Should be `--placement-host-address`, instead of `--placement-address`

Reply:
@mtaghavi2005 @thangchung I incorporated this change here. Should now support Dapr rc1. :)